dataclients/kubernetes/kube.go

```diff
@@ -974,6 +991,13 @@ func (c *Client) ingressToRoutes(state *clusterState) ([]*eskip.Route, error) {
 	return routes, nil
 }
 
+func defaultFiltersOf(service string, namespace string, defaultFilters map[resourceId]string) (string, bool) {
+	if filters, ok := defaultFilters[resourceId{name: service, namespace: namespace}]; ok {
```

Comment:
I think this should work and would be the same:
```
return defaultFilters[resourceId{name: service, namespace: namespace}]
```

Reply:
Looks like.

Reply:
It doesn't if we not explicitly say that we expect 2 return values.